src/ert/gui/main.py

```diff
@@ -88,11 +88,11 @@ def _start_initial_gui_window(args, log_handler):
         ]
     except ConfigValidationError as error:
         messages.append(str(error))
-        logger.info("Error in config file shown in gui: '%s'", error)
+        logger.info("Error in config file shown in gui: '%s'", str(error))
```

Comment:
Is there a special reason why you did not change:
`logger.info("Error in config file shown in gui: '%s'", str(error))`
to 
`logger.info(f"Error in config file shown in gui: '{error}'")`

Reply:
It does mean slightly different things to the azure logger. In the second case you just get a message, in the other case the data is send separately.

Reply:
Ok, and why is it preferable to send data separately?

Reply:
There is a limit on the message length, and the data is treated as json.